features/upgrade/sdn/policy-upgrade.feature

```diff
@@ -78,3 +78,116 @@ Feature: SDN compoment upgrade testing
       | curl | -s | --connect-timeout | 5 | <%= cb.pod2ip %>:8080 |
     Then the step should fail
     And the output should not contain "Hello"
+
+
+  # @author asood@redhat.com
+  @admin
+  @upgrade-prepare
+  Scenario: Check the namespace networkpolicy for an application works well after upgrade - prepare
+    Given I switch to cluster admin pseudo user
+    When I run the :new_project client command with:
+      | project_name | test1 |
+    Then the step should succeed
+    When I run the :new_project client command with:
+      | project_name | test2 |
+    Then the step should succeed
+
+    When I use the "test1" project
+    Given I obtain test data file "networking/list_for_pods.json"
+    And I run the :create client command with:
+      | f | list_for_pods.json |
+    Then the step should succeed
+    Given 2 pods become ready with labels:
+      | name=test-pods |
+    And evaluation of `pod(1).ip` is stored in the :pod2ip clipboard
+    And evaluation of `pod(0).name` is stored in the :pod1 clipboard
+
+    Given I obtain test data file "rc/idle-rc-1.yaml"
+    And I run the :create client command with:
+      | f | idle-rc-1.yaml |
+    Then the step should succeed
+    Given 2 pods become ready with labels:
```

Comment:
we can update the replicas to 1 if only need one pod. 
```
    When I run oc create over "list_for_pods.json" replacing paths:
      | ["items"][0]["spec"]["replicas"] | 1 |
```

Reply:
Sure this can be changed for namespace policy-upgrade2.

Reply:
Reduced # of pods in policy-upgrade2 namespace to 2 pods (1 test-pods and 1 hello-idle) and in policy-upgrade1 to 3 pods (2 test-pods and 1 hello-idle)